src/app/portlets/dot-edit-page/content/dot-edit-content.component.scss

```diff
@@ -23,6 +23,15 @@ dot-whats-changed,
     display: flex;
     justify-content: space-between;
     padding: $basic-padding * 2;
+
+    & > * {
+        display: flex;
```

Comment:
What is this doing?

Reply:
is making this   https://github.com/dotCMS/core-web/blob/bea62b9a6f5aeeea2cce993604a1a51bff5aa5ce/src/app/portlets/dot-edit-page/content/dot-edit-content.component.html#L12  display flex. to show `<dot-global-message>` correctly